### PR TITLE
[DCFS-7] Style the Product Details Page Using CSS

### PR DESCRIPTION
**Pull request by kemysola**
## Task: `Product Page`
- 
Screen Shot 2021-03-06 at 2 28 38 PM

@
Screen Shot 2021-03-06 at 2 29 13 PMt

Screen Shot 2021-03-06 at 2 30 41 PM

@adaugochi and @ToluAkin .Kindly approve.Thank you

### CODE REVIEW

**Comment by adaugochi** [comment]
Kindly improve on writing commit messages 
image
this doesn't matter sense. You do that quiet often. It won't be accepted next time

File: submissions/adekemiAwomolo/html-forms/product.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/adekemiAwomolo/html-forms/product.html:1]: Kindly work on your existing product page you have created in your `html` folder

**Review by adaugochi — CHANGES_REQUESTED** [review]
All images should be placed in your `images` folder that is in your `html` folder.

Nice design you have there. Well done

File: submissions/adekemiAwomolo/html-forms/product.html (hunk 1 above)
Comment (kemysola) [reply submissions/adekemiAwomolo/html-forms/product.html]: @adaugochi Good afternoon ma, i am so sorry about my commit messages ma, i have worked on all the corrections you asked ma.Kindly review ma, thank you very much ma.

**Review by adaugochi — APPROVED** [review]

**Pull request merged**